x-pack/legacy/plugins/maps/public/connected_components/map/mb/view.js

```diff
@@ -436,6 +435,11 @@ export class MBMapContainer extends React.Component {
     });
   }
 
+  _loadMakiSprites() {
+    // TODO Should we preface sprite ids in @elastic/maki? e.g. `__kbn__airfield-15`
```

Comment:
Agreed, name spacing maki sprites is a good idea but how about putting that into a separate PR?